src/core.rs

```diff
@@ -211,16 +233,18 @@ pub struct Core {
     our_connection_info_map: LruCache<PublicId, OurConnectionInfo>,
     their_connection_info_map: LruCache<PublicId, TheirConnectionInfo>,
     get_request_count: usize,
```

Comment:
Thanks for cleaning up the mess with the statistics! Shouldn't `get_request_count` also go into `DebugStats`?